### ISSUE TITLE
Broken AUR

### ISSUE THREAD

**Issue by hvitoi** [CheesyPhoenix/hyprland-relative-workspace#3]
Seems that the compiled binary has been removed

```shell
install: cannot stat 'hyprland-relative-workspace': No such file or directory
==> ERROR: A failure occurred in package().
    Aborting...
error: failed to build 'hyprland-relative-workspace-bin-1.1.7-1': 
error: packages failed to build: hyprland-relative-workspace-bin-1.1.7-1
```

**Comment by CheesyPhoenix** [comment]
Yeah, i messed up the PKGBUILD file. Will (try to) fix this later today.

In the mean time, use an older version, or install from crates.io.

**Comment by CheesyPhoenix** [comment]
Should now be fixed as of version ``1.1.8``
